src/ArcGISRuntime.Viewers.All.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
Remember and roll this back to version 14 Visual studio 2015. Rather than checking it in as a 2017 solution